test/test_server.py

```diff
@@ -226,19 +226,6 @@ def test_get_value_at_indices():
     numpy.testing.assert_allclose(server.getValueAtIndices(request, None).values_double.values, values.flatten())
 
 
-def test_get_vals_indices_2d():
```

Comment:
This test is obsolete?

Reply:
the heat model uses a [flat array now](https://github.com/csdms/bmi-example-python/blob/e6b1e9105daef44fe1f0adba5b857cde1bbd032a/tests/test_get_value.py#L56) so I thought indeed the test was obsolete.